dashboard/tests/unit/jobs/test_csvmapper.spec.ts

```diff
@@ -152,15 +152,15 @@ describe("Test CSV Mapper", () => {
 
     const inverter = testJob.definition.system_definition.inverters[0];
     const array = inverter.arrays[0];
-    const expectedMetadata = {
```

Comment:
doesn't really matter, but was this one (and ln 163) supposed to change?

Reply:
No, thanks.